npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater.rb

```diff
@@ -442,6 +444,20 @@ def git_dependencies_to_lock
           @git_dependencies_to_lock
         end
 
+        def lock_deps_with_latest_reqs(content)
```

Comment:
Let's add a comment here explaining why we're doing this (I'm definitely going to forget..), something like:

```ruby
# When a package.json version requirement is set to `latest` npm will always try to update
# these dependencies when doing an npm install regardless of lockfile version. Prevent any 
# unrelated updates by changing the version requirement to `*` while updating the lockfile.

Reply:
Good idea!